src/registration/tests/test_supplements.py

```diff
@@ -51,10 +51,14 @@ def setUp(self):
         from registration.tests.utils import clear_all_meta_caches
         # recall syncdb
         recall_syncdb()
-        # clear caches
-        clear_all_meta_caches()
-        from django.db.models import loading
-        loading.cache.loaded = False
+        try:
+            # clear caches
+            clear_all_meta_caches()
+            from django.db.models import loading
+            loading.cache.loaded = False
+        except ImportError:
+            # In Django1.9, django.db.models.loading is removed
+            pass
```

Comment:
From the very beginning, Do we need these setup phases?
It seems to pass all tests in all Django versions without these initialise.